packages/frontend-v2/utils/theme.ts

```diff
@@ -76,8 +81,29 @@ const space = {
   "2xl": "3rem",
 };
 
+/**
+ * Customized component styles can be configured and passed to the theme provider
+ */
+const components = {
+  Button,
+}
+
+// Allows us to reuse common styles across pages
+const layerStyles = {
```

Comment:
[this](https://chakra-ui.com/docs/styled-system/features/text-and-layer-styles) allows styling combinations to be reused, but i think putting this & other reusable combinations here could cause the theme file to get cluttered. if this is the right way to go, it might be worth it to place these objects in a separate file and import them? or it might be better to make this its own custom component... tbh im not sure lol

Reply:
I love the thought process here! I do feel that the flex header container is better suited as a react component though since header container styles are specific to headers for pages. A single page at most has 1 header and a header is always a Flex component.

Layered styles are nice when we have a group of styles that we would like to use for across multiple components. For example, a set of styles for "selected" that could be applied to Boxes, Buttons etc. would be an awesome layered style.

For here though, we can have a header container component. It can go in the components directory since it seems reusable :)

Reply:
ok sounds good!!